frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -357,6 +357,9 @@ const Card: React.FC<CardProps> = ({ panel }) => {
   }
 
   const panelClassModifier = getPanelClassModifier(panel);
```

Comment:
Nit: Since `panelClassModifier` is not used in the early return case, better to move it after the early return.

Reply:
Nit, uh, combed? All set.